contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -141,6 +154,18 @@ def validate_configuration(
                 f"{'foreign_table_key_column ' if 'foreign_table_key_column' not in configuration.kwargs else ''}"
             )
 
+    def _generate_partial_unexpected_counts(self, unexpected_list):
+        """Generate partial_unexpected counts using logic borrowed from _format_map_output() in expectations.py"""
+        return [
+            {"value": key, "count": value}
+            for key, value in sorted(
+                Counter(unexpected_list).most_common(
```

Comment:
This feels a little complex - could you explain what’s happening here?

Reply:
This is finding the most common unexpected values. Will is adding to docstring.